### DISCUSSION TITLE
Distinguish the firing alert that was or wasn't muted

### DISCUSSION THREAD

**Discussion by freak12techno** [grafana/grafana#41097] [Alerting]
**What would you like to be added**: The firing alert that was muted and the firing alert that wasn't muted should be displayed differently

**Why is this needed**: For now, there are several places (specifically, but probably not limited to: alerting list, the "Alerts list" component") where you can see the list of alerts. You can understand by the color/label if the the alert is okay, pending or fired. But once fired, you cannot visually distinguish the muted alert and the not muted one.
This may be fixed by adding a different color/label text for muted alerts (for example, gray and "Silenced"), this way the user would easily see if the alert was muted.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
